portfolio/src/main/webapp/script.js

```diff
@@ -45,11 +50,54 @@ async function getRandomQuoteUsingAsyncAwait() {
 }
 
 async function loadTasks() {
-    let list = await fetch('/list-tasks').then(response => response.json());
+    let response = await fetch('/list-tasks');
+    let list = await response.json();
     for(c in list){ 
         let reviewsElem = document.getElementById("reviews");
         let commentElem = document.createElement("DIV");
-        commentElem.appendChild(document.createTextNode(list[c].type+" "+list[c].description));
+        commentElem.appendChild(document.createTextNode(list[c].email+": "+list[c].description));
         reviewsElem.appendChild(commentElem);
     }
-}
\ No newline at end of file
+    document.getElementById("showCQC").onclick = null;
+}
+async function loginStatus() {
+    let login = await fetch('/status');
+    let status = await login.json();
+    console.log(status);
+    if (status==0){
+        document.getElementById("cqcForm").style.display="none";
+    }else{
+        document.getElementById("cqcForm").style.display="block";
```

Comment:
2 comments:

1. Spacing between the else statement: `} else {`
2. You can reduce the duplicate code here!

```
let display = "block";
if (status == 0) {
    display = "none";
}
document.getElementById("cqcForm").style.display = display;
```